plugins/symbol-processor/src/test/kotlin/org/jetbrains/dataframe/ksp/DataFrameJdbcSymbolProcessorTest.kt

```diff
@@ -97,20 +97,20 @@ class DataFrameJdbcSymbolProcessorTest {
                     SourceFile.kotlin(
                         "MySources.kt",
                         """
-                @file:ImportDataSchema(name = "Customer", path = "123")
-                
-                package test
-                
-                import org.jetbrains.kotlinx.dataframe.annotations.ImportDataSchema
-                import org.jetbrains.kotlinx.dataframe.annotations.JdbcOptions
-                import org.jetbrains.kotlinx.dataframe.api.filter
-                import org.jetbrains.kotlinx.dataframe.DataFrame
-                import org.jetbrains.kotlinx.dataframe.api.cast
-                import java.sql.Connection
-                import java.sql.DriverManager
-                import java.sql.SQLException
-                import org.jetbrains.kotlinx.dataframe.io.readSqlTable
-                import org.jetbrains.kotlinx.dataframe.io.DatabaseConfiguration
+                        @file:ImportDataSchema(name = "Customer", path = "123")
+                        
+                        package test
+                        
+                        import org.jetbrains.kotlinx.dataframe.annotations.ImportDataSchema
```

Comment:
Did you do it manually or with automated refactoring? I had some troubles with automated

Reply:
manually, unfortunately. But it makes sense that it isn't done automatically. Since it's a String you could mean anything, so it's good IntelliJ doesn't change it.